README.md

```diff
@@ -700,6 +700,64 @@ Get builds that belong to this event
 model.getBuilds()
 ```
 
+### Template Model
+```js
+'use strict';
+const Model = require('screwdriver-models');
+const factory = Model.TemplateFactory.getInstance({
+    datastore
+});
+const config = {
+    name: 'testTemplate',
```

Comment:
@tkyi @FenrirUnbound I am not sure if we should enforce platform/step or let the platform owner to decide what is the proper format. If `platform/step` then we need to change the `data-schema`